webpage/grammar_checker.py

```diff
@@ -38,11 +38,11 @@ def correction(self,word):
     # generate possible spelling correciton for word
     def candidates(self,word):
         if len(word)<5:
-            return self.known([word]) or self.known(self.edits1(word)) or self.known(self.edits2(word)) or self.known(self.edits3(word)) or set([word])
-        elif len(word)>=5 and len(word)<13:
             return self.known([word]) | self.known(self.edits1(word)) or self.known(self.edits2(word)) or self.known(self.edits3(word)) or set([word])
```

Comment:
Candidate words for short word (length<5) should be the original word + valid words with one edit distance.

Reply:
so instead of:
def candidates(self,word):
        if len(word)<5:
            return self.known([word]) or self.known(self.edits1(word)) or self.known(self.edits2(word)) or set([word])
        elif len(word)>=5 and len(word)<13:
            return self.known([word]) | self.known(self.edits1(word)) or self.known(self.edits2(word))  or set([word])
        else:
            return self.known([word]) | self.known(self.edits1(word)) | self.known(self.edits2(word)) or set([word])

do?
def candidates(self,word):
        if len(word)<5:
            return self.known([word]) | self.known(self.edits1(word)) or self.known(self.edits2(word)) or set([word])
        elif len(word)>=5 and len(word)<13:
            return self.known([word]) | self.known(self.edits1(word)) | self.known(self.edits2(word))  or set([word])
        else:
            return self.known([word]) | self.known(self.edits1(word)) | self.known(self.edits2(word)) or self.known(self.edits3(word)) or set([word])